scripts/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM conda/miniconda3
+
+MAINTAINER Maxim Kochurov <maxim.v.kochurov@gmail.com>
```

Comment:
Maintainer is deprecated. Suggestion for new pattern is in the docs

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated